### ISSUE TITLE
Change arrows (request/transfer) 

### ISSUE THREAD

**Issue by marqmarti** [walleth/walleth#177]
Arrows in the main tab for requesting or transferring tokens look very confusing 
As they are both green and point strange directions. They are not recognizable at first sight that they mean "requests" or "transfer tokens". It took more than 15 minutes to me to realize it. 

Arrows should be of different colors and point better angles. It would be nice that "request" would have an arrow pointing down and "transfer" (or, even better "send") would have an arrow pointing up.

**Comment by friedger** [comment]
I agree. I think the idea is that the arrows indicate value going in and value going out. And at the same time try to link to the two columns below.
* Introducing headers for the two columns of transactions would simply things.
* Green for requests and red for sending could be better indication of the arrow actions.

**Comment by ligi** [comment]
Sorry but I disagree here. 
Regarding the colors:
 - green is our accent color and should be used for things like this
 - I really do not want to go the red/green route for a lot of reasons ( e.g. there is red/green blindness, also red/green has positive/negative associations that I really do not want here)

Regarding the directions:
 - If we change to up/down I would go for the opposite (receive up, send down) as I want to have the arrows point to the value in the correct direction - everything else I see as really confusing. The receive should point to the value and send/transfer away from the value.

Perhaps this would be a compromise we can go if you really want up/down arrows:

![memo20180210_122122](https://user-images.githubusercontent.com/111600/36061649-3ddbae38-0e5d-11e8-9e1d-5b3a2077d886.jpg)

This way we loose some height (hence I was going with the buttons left and right of the value. On the other and it gives us some more space for the value. What do you think?
